packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -28,7 +28,7 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<HTMLDivEle
   /**
    * Callback issued when the selected option changes.
    */
-  onChange?: (event: React.FormEvent<HTMLDivElement>, option: IDropdownOption, index?: number) => void;
+  onChange?: (event: React.FormEvent<HTMLDivElement>, option?: IDropdownOption, index?: number) => void;

```

Comment:
could this be considered breaking? being required means they can rely on the value being there, i.e. they wouldn't need to check.

Reply:
probably has some implications of null checks - issue is that as is it wouldn't compile :(

Reply:
Note: the Basic example has code using onChange and I didn't have to change it because event handlers usually have to declare their params anyway...